packages/utils/src/commonwell/enhance-coverage.ts

```diff
@@ -0,0 +1,61 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+// keep that ^ on top
+import { CodeChallenge } from "@metriport/core/domain/auth/code-challenge";
+import { cookieFromString } from "@metriport/core/domain/auth/cookie-management/cookie-manager";
+import { CookieManagerInMemory } from "@metriport/core/domain/auth/cookie-management/cookie-manager-in-memory";
+import { CommonWellManagementAPI } from "@metriport/core/external/commonwell/management/api";
+import {
+  SessionManagement,
+  SessionManagementConfig,
+} from "@metriport/core/external/commonwell/management/session";
+import { getEnvVar, getEnvVarOrFail } from "@metriport/core/util/env-var";
+import * as readline from "readline-sync";
+
+/**
+ * Script to run on local environment the code that enhances coverage @ CommonWell.
```

Comment:
Is this still a TODO? The code in `main()` doesn't seem to match the description

Reply:
Good catch, I'll update this and rename the file